tests/rt_utils.sh

```diff
@@ -387,13 +387,7 @@ rocoto_create_compile_task() {
     echo "  </metatask>" >> $ROCOTO_XML
   fi
 
-  # serialize WW3 builds. FIXME
-  DEP_STRING=""
-  if [[ ${COMPILE_PREV_WW3_NR} != '' ]]; then
-      if [[ ${MAKE_OPT^^} =~ "-DAPP=ATMW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=S2SW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=HAFSW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=HAFS-ALL" ]] ; then
-          DEP_STRING="<dependency><taskdep task=\"compile_${COMPILE_PREV_WW3_NR}\"/></dependency>"
-      fi
-  fi
+  ## serialize WW3 builds. FIXME
```

Comment:
Can we remove this comment?

Reply:
Removed.

Reply:
If it's not too late for this PR, `COMPILE_PREV_WW3_NR` can also be removed from `rt.sh` and `opnReqTest`

Reply:
@DusanJovic-NOAA would these be the proper removals: 

```
Jessica.Meixner@hfe10:/scratch2/NCEPDEV/climate/Jessica.Meixner/PR_WW3/ufs-cmake$ git diff 
diff --git a/tests/opnReqTest b/tests/opnReqTest
index 78e47ac8..28f32f72 100755
--- a/tests/opnReqTest
+++ b/tests/opnReqTest
@@ -531,7 +531,6 @@ mkdir -p ${RUNDIR_ROOT}
 if [[ $ECFLOW == true ]]; then
   ECFLOW_RUN=${PATHRT}/ecflow_opnReqTest_run
   ECFLOW_SUITE=opnreqtest_$$
-  COMPILE_PREV_WW3_NR=
   rm -rf ${ECFLOW_RUN}
   mkdir -p ${ECFLOW_RUN}/${ECFLOW_SUITE}
   cp head.h tail.h ${ECFLOW_RUN}
diff --git a/tests/rt.sh b/tests/rt.sh
index c5ab9a74..aabdc9a3 100755
--- a/tests/rt.sh
+++ b/tests/rt.sh
@@ -522,7 +522,6 @@ source default_vars.sh
 JOB_NR=0
 TEST_NR=0
 COMPILE_NR=0
-COMPILE_PREV_WW3_NR=''
 rm -f fail_test* fail_compile*
 
 export LOG_DIR=${PATHRT}/log_$MACHINE_ID
@@ -741,10 +740,6 @@ EOF
       BL_SUFFIX=""
     fi
 
-    if [[ ${MAKE_OPT^^} =~ "-DAPP=ATMW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=S2SW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=HAFSW" ]] || [[ ${MAKE_OPT^^} =~ "-DAPP=HAFS-ALL" ]] ; then
-       COMPILE_PREV_WW3_NR=${COMPILE_NR}
-    fi
-
     continue
 
   elif [[ $line == RUN* ]] ; then
```

Reply:
Yes.